### ISSUE TITLE
Implement immutable.js features

### ISSUE THREAD

**Issue by czerwinskilukasz1** [xFAANG/askql#130]
Make immutable.js accessible from AskScript. This would greatly enhance the language.

**Comment by mhagmajer** [comment]
Yes, good idea. Since Immutable.js is JavaScript specific, let's integrate it into another JavaScript specific package - https://github.com/xFAANG/askql/issues/61
